app/javascript/App/bundles/CoinShow/HistoricalPriceDataTable.tsx

```diff
@@ -141,17 +139,25 @@ class HistoricalPriceDataTable extends React.Component<Props, State> {
   }
 
   public parseData(d: RawPriceData): PriceData {
```

Comment:
You can do `parseData = (d: RawPriceData): PriceData => {` to avoid explicitly binding to `this`